src/styles/blocks/card.css

```diff
@@ -0,0 +1,120 @@
+.card {
+  display: flex;
+  justify-content: center;
+  align-content: center;
+  align-items: center;
+  flex-direction: column;
+
+  width: 198px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__img-link {
+  margin-top: 32px;
+  margin-bottom: 40px;
+}
+
+.card__img {
+  display: block;
+  margin: 0 auto;
+  height: 134px;
+  width: 160px;
+}
+
+.card__name-link {
+  text-decoration: none;
+  color: #060b35;
+
+  width: 166px;
+  margin-bottom: 4px;
+
+}
+
+.card__name-link_text {
+  margin: 0;
+
+  font-size: 12px;
```

Comment:
you have default font size for whole page 12px, so it's not necessary to put it everywhere where we need it